ui/seed_backup_page.go

```diff
@@ -0,0 +1,538 @@
+	button.skin.Background = pg.theme.Color.Hint
+	button.skin.TextSize = unit.Dp(18)
+	if sg.selected == buttonIndex {
+		button.skin.Background = pg.theme.Color.Primary
+	}
+	layout.Inset{Right: unit.Dp(15), Left: unit.Dp(15)}.Layout(pg.gtx, func() {
+		button.skin.Layout(pg.gtx, sg.buttons[buttonIndex].button)
+	})
+}
+
+func (pg *backupPage) verifyCheckBoxes() bool {
+	for _, cb := range pg.checkBoxWidgets {
+		if !cb.Checked(pg.gtx) {
+			return false
+		}
+	}
+	return true
+}
+
+func (pg *backupPage) randomSeeds() []string {
+	var randomSeeds []string
+
+	for i := 0; i < 3; i++ {
+		random := rand.Intn(len(pg.allSuggestions))
+		randomSeeds = append(randomSeeds, pg.allSuggestions[random])
+	}
+	return randomSeeds
+}
+
+func (pg *backupPage) populateSuggestionSeeds() {
+	rand.Seed(time.Now().Unix())
+
+	for k := range pg.seedPhrase {
+		seeds := pg.randomSeeds()
+		s := pg.suggestions[k]
+		for i := range s.buttons {
+			s.buttons[i].skin.Text = seeds[i]
+		}
+		s.buttons[rand.Intn(len(seeds))].skin.Text = pg.seedPhrase[k]
+	}
+}
+
+func (pg *backupPage) updateViewTexts() {
+	switch pg.active {
+	case infoView:
+		pg.title.Text = "Keep in mind"
+		pg.action.Text = "View seed phrase"
+	case seedView:
+		pg.title.Text = "Write down seed phrase"
+		pg.action.Text = "I have written down all 33 words"
+		pg.steps.Text = fmt.Sprintf("Steps %d/2", seedView-1)
+		pg.instruction.Text = "Write down all 33 words in the correct order"
+	case verifyView:
+		pg.title.Text = "Verify seed phrase"
+		pg.action.Text = "Verify"
+		pg.steps.Text = fmt.Sprintf("Steps %d/2", verifyView-1)
+		pg.instruction.Text = "Select the correct words to verify"
+	case successView:
+		pg.action.Text = "Back to Wallets"
+	}
+}
+
+func (pg *backupPage) clearError() {
+	time.AfterFunc(time.Second*3, func() {
+		pg.error = ""
+	})
+}
+
+func checkSlice(s []string) bool {
+	for _, v := range s {
+		if v == "-" {
+			return false
+		}
+	}
+	return true
+}
+
+func (pg *backupPage) resetPage(c pageCommon) {
+	*c.page = PageWallet
+	pg.active = infoView
+	pg.seedPhrase = []string{}
+	pg.selectedSeeds = make([]string, 33)
+	for _, cb := range pg.checkBoxWidgets {
+		cb.SetChecked(false)
+	}
+	for i := range pg.suggestions {
+		pg.suggestions[i].selected = -1
+	}
+	for i := range pg.selectedSeeds {
+		pg.selectedSeeds[i] = "-"
+	}
+	pg.updateViewTexts()
+}
+
+func (pg *backupPage) handle(c pageCommon) {
+	if pg.backButtonWidget.Clicked(pg.gtx) {
+		pg.resetPage(c)
+	}
+
+	if pg.actionWidget.Clicked(pg.gtx) && pg.verifyCheckBoxes() {
+		switch pg.active {
+		case infoView:
+			s := pg.wal.GetWalletSeedPhrase(pg.info.Wallets[*c.selectedWallet].ID)
+			pg.seedPhrase = strings.Split(s, " ")
+			pg.populateSuggestionSeeds()
+			pg.active++
+		case verifyView:
+			if !checkSlice(pg.selectedSeeds) {
+				return
+			}
+			s := strings.Join(pg.selectedSeeds, " ")
+			if !dcrlibwallet.VerifySeed(s) {
+				pg.error = "Failed to verify. Please go through every word and try again."
```

Comment:
The error message here and on line 515 are repeating. Kind initialize a variable for this.